students/ericstreit/session03/mailroom.py

```diff
@@ -32,45 +32,54 @@ def mainmenu():
             print("Goodbye!")
             break
         else:
-             print("Please enter 'S', 'L' or 'Q'")
+             print("\nPlease enter 'S', 'R' or 'Q'")
 
 def thankyou():
-    """This is the thank you menu"""
-    name = input("Please enter the name of the donor or type 'list' to list current donors.")
+    """This is the thank you menu that routes to the listuser or donor_update function"""
+    print("Please enter the name of the donor or type 'list' to list current donors.")
+    name = input(": ")
     if name == "list":
-        listusers()
+        list_users()
     else:
-        donorupdate(name)
+        donor_update(name)
+
+def list_users():
+    """A simple function that lists donor names"""
+    print("\nDonors\n", "{:-<60}".format(""))
+    for i in donors:
+        print(i[0])
 
 def compose_email(name, amount):
     """This function will compose the thank you email"""
     print("\n\nEMAIL SENT:\n'Thank you, {}, for your generous contribution of ${}! We will be sure to ask you for more money again soon.'\n\n".format(name, amount))
```

Comment:
a good idea to break this down into more than one line -- not great to have a line this long...

you could even define the letter template outside his function altogether -- maybe with a triple quoted string.